src/components/button-group/ButtonGroup.scss

```diff
@@ -60,32 +65,37 @@
         }
     }
 
-    > .btn:first-child {
+    > .btn:first-child,
+    > .bdl-targeted-click-through > .btn:first-child {
         border-top-left-radius: 4px;
         border-bottom-left-radius: 4px;
     }
 
-    > .btn:last-child {
+    > .btn:last-child,
+    > .bdl-targeted-click-through > .btn:last-child {
         border-top-right-radius: 4px;
         border-bottom-right-radius: 4px;
     }
 
-    > .btn.is-selected {
+    > .btn.is-selected,
+    > .bdl-targeted-click-through > .btn.is-selected {
         z-index: 2; /* place on top of siblings */
         color: $bdl-gray-80;
         background-color: $bdl-gray-10;
         border-color: $bdl-gray-62;
         box-shadow: none;
     }
 
-    > .btn:focus {
+    > .btn:focus,
+    > .bdl-targeted-click-through > .btn:focus {
         z-index: 3; /* place on top of all other buttons for accessibility */
     }
 
     &.bdl-is-disabled,
     &.is-disabled {
         > .btn,
-        > .bdl-Button {
+        > .bdl-Button,
+        > .bdl-targeted-click-through > .btn:focus {
```

Comment:
It looks like this rule only applies to `.btn`, not `.bdl-Button`. Also, should this rule have `:focus` attached? It's not present in the rule it modifies.

Reply:
oops, good catch, I will remove the focus/

Reply:
I'm not sure what you mean by `It looks like this rule only applies to .btn, not .bdl-Button`?